packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -0,0 +1,69 @@
+import { BaseInput } from '../BaseInput';
+import type { BaseInputProps } from '../BaseInput';
+import { HiddenInput } from './HiddenInput';
+import type { IconComponent } from '~components/Icons';
+import { ChevronDownIcon, ChevronUpIcon } from '~components/Icons';
+import Box from '~components/Box';
+type SelectInputProps = Pick<
+  BaseInputProps,
+  | 'label'
+  | 'labelPosition'
+  | 'necessityIndicator'
+  | 'validationState'
+  | 'helpText'
+  | 'errorText'
+  | 'successText'
+  | 'placeholder'
+  | 'name'
+  | 'isDisabled'
+  | 'isRequired'
+  | 'prefix'
+  | 'suffix'
+  | 'autoFocus'
+> & {
+  icon?: IconComponent;
+};
+
+const SelectInput = (props: SelectInputProps): JSX.Element => {
+  /**
+   * @TODO handle expanded state with Dropdown
+   */
+  const isPopupExpanded = false;
+
+  const { icon, ...baseInputProps } = props;
+
+  return (
+    <Box position="relative">
+      {/* @TODO Use this for form submissions */}
+      <HiddenInput
```

Comment:
It'll also need tabIndex={-1} so that keyboard doesn't get focus to the hidden input.

Reply:
ops yes good catch